applications/nrf_desktop/README.rst

```diff
@@ -1512,6 +1512,11 @@ For example, the following line will start the upload of the new image to the de
 
 ----
 
+.. options-from-kconfig::
+   :show-type:
+
+----
```

Comment:
Please do not add section dividers like this anymore. They are being now automatically added to h2 headings, as per https://github.com/nrfconnect/sdk-nrf/pull/3036

Reply:
removed